helios-tools/src/main/java/com/spotify/helios/cli/CliParser.java

```diff
@@ -297,10 +299,14 @@ public boolean getJson() {
           .setDefault(System.getProperty("user.name"))
           .help("username");
 
-      googleCredentialsArg = addArgument("--google-application-default-credentials")
+      useGoogleCredentialsArg = addArgument("--use-google-credentials")
           .type(Boolean.class)
           .setDefault(true)
-          .help(SUPPRESS);
+          .help("enable Google Credentials to supply an access token");
```

Comment:
would be helpful to mention what happens if `--google-credentials FILE` isn't passed